src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURE_COUNT = 5;
+
+    protected Random random = new Random();
+
+    public Figure getRandomFigure() {
+        switch (random.nextInt(FIGURE_COUNT)) {
+            case 0:
+                return new Circle();
+            case 1:
+                return new Rectangle();
+            case 2:
+                return new Square();
+            case 3:
+                return new IsoscelesTrapezoid();
+            case 4:
+                return new RightTriangle();
+            default:
+        }
+        return null;
+    }
```

Comment:
we should change it after other comments will be fixed
also, `return null` may be redundant, we can return default figure for the default case